source/conf.py

```diff
@@ -43,6 +47,10 @@
 
 .. |version| replace:: {version}
 
+.. |platform_version| replace:: {platformVersion}
+
+.. |fabric_platform_version| replace:: {fabricPlatformVersion}
```

Comment:
Maybe rename the plain `version` to `api_version` to properly differentiate from the platform versions?